numpy/random/_generator.pyx

```diff
@@ -396,6 +396,21 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized beta distribution.
 
+        Examples
+        --------
+        Given a sample of `n` i.i.d. random variables drawn from a uniform 
+        distribution on `[0, 1]`, it is known that the distribution of the 
+        `k`-th smallest value is `Beta(k, n+1-k)`.
+
+        In the case of `n = 100` and `k = 5`, we can simulate this distribution
+        as follows:
+
+        >>> rng = np.random.default_rng()
+        >>> n, k = 100, 5
+        >>> size = 3
+        >>> rng.beta(k, n+1-k, size)
```

Comment:
```suggestion
        >>> rng.beta(k, n+1-k, size=size)
```

Nitpick: I have a mild preference for passing `size=` as a keyword argument to clearly distinguish it from the other distribution parameters. It matters less in this particular instance because you are using sensibly-named variables instead of literal values, but keep that in mind if you refine this example some more.